docs/source/kale.pipeline.rst

```diff
@@ -46,6 +46,14 @@ kale.pipeline.multi\_domain\_adapter module
    :undoc-members:
    :show-inheritance:
 
+kale.pipeline.multi\_modal module
```

Comment:
We have agreed to use multimodal rather than multi-modal. 
What other module ends with module please? I suggest you to review your own code first before asking me to review.